src/util.rs

```diff
@@ -58,11 +58,21 @@ fn test_parse_size() {
 }
 
 pub fn read_passphrase() -> io::Result<String> {
+    // The environment variable should only be used for testing
+    if let Ok(pass) = env::var("RDEDUP_PASSPHRASE") {
+        printerr!("Using passphrase set in RDEDUP_PASSPHRASE\n");
+        return Ok(pass);
+    }
     printerr!("Enter passphrase to unlock: ");
     rpassword::read_password()
 }
 
 pub fn read_new_passphrase() -> io::Result<String> {
+    // The environment variable should only be used for testing
+    if let Ok(pass) = env::var("RDEDUP_PASSPHRASE") {
```

Comment:
Ok I am work on another request after to read from a file then =) I am sure someone already requested that.